web/auth/interceptors/tokens.go

```diff
@@ -0,0 +1,60 @@
+package interceptors
+
+import (
+	"context"
+	"strings"
+
+	"github.com/autograde/quickfeed/web/auth"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+func ValidateToken(logger *zap.Logger, tokens *auth.TokenManager) grpc.UnaryServerInterceptor {
```

Comment:
Seems like we only use the `SugaredLogger`, so we should pass that in instead. (Same for all inceptors I think)